dockers/docker-fpm-frr/base_image_files/TS

```diff
@@ -4,10 +4,11 @@
 [ -f /etc/sonic/sonic-environment ] && . /etc/sonic/sonic-environment
 
 PLATFORM=${PLATFORM:-`sonic-cfggen -H -v DEVICE_METADATA.localhost.platform`}
-type=`sonic-db-cli CONFIG_DB hget 'DEVICE_METADATA|localhost' 'type'`
+subtype=`sonic-db-cli CONFIG_DB hget 'DEVICE_METADATA|localhost' 'subtype'`
```

Comment:
subtype may not be defined for all sonic SpineRouter sku's?

Reply:
This is mainly used for reliable TSA from supervisor on modular chassis. 
subtype is being used to check if the device is a linecard (UpstreamLC/DownstreamLC). If not set, assumption is that it is not a linecard.